swagger.yml

```diff
@@ -12,124 +12,137 @@ info:
         "503":
-          $ref: '#/responses/InternalServerError'
+          $ref: "#/responses/InternalServerError"
   /user:
     get:
       tags:
-      - "user"
+        - "user"
       summary: "Get the details of the users registered in the system"
       description: "This can be done only by an authenticated user"
       operationId: "user"
       produces:
-      - "application/json"
+        - "application/json"
       parameters:
-      - name: "username"
-        in: "query"
-        description: "The user name for login"
-        required: true
-        type: "string"
-      - name: "password"
-        in: "query"
-        description: "The password for login in clear text"
-        required: true
-        type: "string"
+        # - name: "username"
+        #   in: "query"
+        #   description: "The user name for login"
+        #   required: true
+        #   type: "string"
+        # - name: "password"
+        #   in: "query"
+        #   description: "The password for login in clear text"
+        #   required: true
+        #   type: "string"
+        - in: "body"
+          name: "body"
+          description: "Get User Details"
+          required: true
+          schema:
+            $ref: "#/definitions/Register"
       responses:
         "200":
           description: "successful operation"
-          schema:
-            $ref: "#/definitions/Register"
+          # schema:
+          # $ref: "#/definitions/Register"
         "401":
-          $ref: '#/responses/UnauthenticatedError'
+          $ref: "#/responses/UnauthenticatedError"
         "403":
-          $ref: '#/responses/UnauthorizedError'
+          $ref: "#/responses/UnauthorizedError"
         "503":
-          $ref: '#/responses/InternalServerError'
+          $ref: "#/responses/InternalServerError"
       security:
         - basicAuth: []
   /orders:
     get:
       tags:
-      - "orders"
+        - "orders"
       summary: "Get the order details"
       description: "This can be done only by an authenticated user"
       operationId: "orders"
       produces:
-      - "application/json"
+        - "application/json"
       parameters:
-      - name: "username"
-        in: "query"
-        description: "The user name for login"
-        required: true
-        type: "string"
-      - name: "password"
-        in: "query"
-        description: "The password for login in clear text"
-        required: true
-        type: "string"
+        - in: "body"
+          name: "body"
+          description: "Created user object"
+          required: true
+          schema:
+            $ref: "#/definitions/order"
+
+        # - name: "username"
+        #   in: "query"
+        #   description: "The user name for login"
+        #   required: true
+        #   type: "string"
+        # - name: "password"
+        #   in: "query"
+        #   description: "The password for login in clear text"
+        #   required: true
+        #   type: "string"
```

Comment:
Dont keep commented values in pull requests. Else we will need a pull request again just to remove them later

Reply:
resolved !!